packages/clients/meldemichel/src/plugins/AfmButton/AfmButton.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-btn
+    v-if="afmUrl"
+    :target="mapStateReady ? '_blank' : ''"
+    color="primary"
+    class="meldemichel-afm-button"
+    :href="mapStateReady ? afmUrl : '#'"
+    large
+    @click="click"
+    @keydown.space="click"
```

Comment:
As `href` and `target` are added to `v-btn`, it is being rendered as an anchor `<a>`. Are you sure, that the intended behaviour applies for anchors as well?

I also find it a bit misleading having the anchor look like a button, but that may just be me 🤷

Reply:
While it's an anchor, it looks like a button, so both interactions might be expected. Allowing space for the interaction should not hurt since it can't be used for scrolling in the `COMPLETE` mode client anyway.

In which way is it misleading? Since it does not just hardlink static content, but continues a process, I deem this correct and in line with the following AfM process.